bin/create-docs-pr.js

```diff
@@ -25,13 +26,8 @@ program.option(
   'Name of changelog(defaults to NEWS.md)',
   DEFAULT_FILE_NAME
 )
-program.option('-f --force', 'bypass validation')
 program.option('--dry-run', 'executes script but does not commit nor create PR')
-program.option(
-  '--repo-path <path',
-  'Path to the docs-website fork on local machine',
-  'docs-website'
-)
```

Comment:
Is there a real need to remove this option? Should the action ever need to be run manually without GHA, this option could still be useful.

Reply:
I've restored it and added a repo-owner option.